tests/store/test_sqlalchemy_store.py

```diff
@@ -15,7 +16,7 @@
 from mlflow.utils.file_utils import TempDir
 
 DB_URI = 'sqlite://'
-ARTIFACT_URI = 'file://fake file'
+ARTIFACT_URI = 'artifact_folder'
 
 
 class TestSqlAlchemyStoreSqliteInMemory(unittest.TestCase):
```

Comment:
Nit: We should probably be using `pytest` here.

Reply:
Do we use `unittest` because it makes it easier to define `tearDown()` routine to purge the Sql store tables?